scripts/logic/merge_status.py

```diff
@@ -1,7 +1,7 @@
 from scripts.utils import is_empty
 
 
-def classif_status(*args):
+def merge_status(*args):
```

Comment:
Why is this script on "logic" folder and not in "custom" folder ?

Reply:
There is logic with the two arguments in this merging script, that's why it fits better here I think. But you make a good point, folders are not well organised right now. However, this does not really impact since all scripts are displayed at the same time on pyrog.